### PR TITLE
Renaming `owner` role name to `admin`

### PR DESCRIPTION
**Pull request by mohammedbabelly20**
From now on the old `owner` team role has been replaced with the `admin` role, so I just updated the docs pages that have the owner role.

### CODE REVIEW

**Review by helinanever — CHANGES_REQUESTED** [review]

File: content/partials/users.md
[the remark this answers is not in the document]
Comment (mohammedbabelly20) [reply content/partials/users.md:1]: Confirmed, we remove its references from other pages.

File: content/rest-api/teams.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Invite a new team member to your team.
 | **Name** | **Type** | **Description**                               |
 |----------| -------- |-----------------------------------------------|
 | `email`  | `string` | **Required.** User email                    |
-| `role`   | `string` | **Required.** Could be `owner` or `developer` |
+| `role`   | `string` | **Required.** Could be `admin` or `developer` |
```
[the remark this answers is not in the document]
Comment (mohammedbabelly20) [reply content/rest-api/teams.md]: My apologies, I thought it was a description of what role you should have before using the API.
No the API hasn't changed, it still expecting `owner` or `developer`.

File: content/rest-api/teams.md [hunk 2]
```diff
@@ -21,9 +21,10 @@ Invite a new team member to your team.
 | `email`  | `string` | **Required.** User email                    |
 | `role`   | `string` | **Required.** Could be `owner` or `developer` |
 
-<br />
+<br>
+
+>`developer` role corresponds to the **Member** role and `owner` role corresponds to the **Admin** role in Codemagic UI.
```
Comment (helinanever) [inline content/rest-api/teams.md:26]: nice 👍

File: content/yaml-running-builds/starting-builds-automatically.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ In order to fully automate your CI/CD pipeline, you can set up automatic build t
 When using the Flutter workflow editor, by clicking **Update Webhook** under the Webhooks tab, Codemagic automatically adds webhooks to the repositories added via GitHub app or from GitHub, GitLab or Bitbucket via the OAuth integration after you have enabled any of the triggers in this section. For repositories added via SSH or HTTP/HTTPS, or if you are configuring your builds using `codemagic.yaml`, you would have to [set up webhooks manually](../building/webhooks). Note that webhook triggers might not be supported for all repository providers.
```
Comment (helinanever) [inline content/yaml-running-builds/starting-builds-automatically.md:14]: The whole Flutter workflow editor section seems irrelevant as we have a separate page about build triggers for workflow editor

File: content/yaml-running-builds/starting-builds-automatically.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ In order to fully automate your CI/CD pipeline, you can set up automatic build t
...
 
 {{<notebox>}}
-**Note:** Webhooks can be clicked to update by the team owner who connected their repository in the Codemagic UI.
+**Note:** Webhooks can be clicked to update by the team admin who connected their repository in the Codemagic UI.
```
Comment (helinanever) [inline content/yaml-running-builds/starting-builds-automatically.md:17]:
```suggestion
**Note:** The team admin who added the repository can update the webhook by clicking the **Update webhook** button in the Codemagic UI.
```

File: content/yaml-running-builds/starting-builds-automatically.md (hunk 3 above)
Comment (mohammedbabelly20) [reply content/yaml-running-builds/starting-builds-automatically.md]: Correct, I removed it but kept the repos that have been added by SSH or for the yaml workflows section.

**Review by helinanever — APPROVED** [review]
LGTM
